### PR TITLE
New permissions file and inclusion of exception manager

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
We have a developer and read access roles sql living in census-rm-ddl that we never updated after the DB split so these needed to be separated to account for the changes

# What has changed
New dev and read permissions scripts for Action DB
Removal of action schemas/tables from original dev/read scripts
Included exception manager schema/tables in original dev/read scripts

# Checklist
Reminder: Make you have run `build_groundzero_ddl.sh` and not manually edited the ground zero DDL.
* [x] Have run automated script, not made manual changes

Reminder: Make sure to version tag this after the PR is merged
* [x] Updated patch number
* [x] Updated version_tag in `ddl_version.sql`
* [x] Updated current_version in `patch_database.py`

# How to test?
Build DDL pod in local environment and run rebuild script to check it all works

# Links
https://trello.com/c/IAmEOpmO/265-fix-database-roles-to-work-after-action-db-split-5

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Fix database roles to work after action DB split (5)](https://trello.com/c/IAmEOpmO/265-fix-database-roles-to-work-after-action-db-split-5)

File: groundzero_ddl/ACTION-ddl_version.sql [hunk 1]
```diff
@@ -4,4 +4,4 @@ CREATE TABLE ddl_version.version (version_tag varchar(256) PRIMARY KEY, updated_
 -- Version and patch number for the current ground zero,
 -- NOTE: These must be updated every time the repo is tagged
 INSERT INTO ddl_version.patches (patch_number, applied_timestamp) VALUES (200, current_timestamp);
-INSERT INTO ddl_version.version (version_tag, updated_timestamp) VALUES ('v1.1.0', current_timestamp);
+INSERT INTO ddl_version.version (version_tag, updated_timestamp) VALUES ('v1.2.0', current_timestamp);
```
Comment (NickGrantONS) [inline groundzero_ddl/ACTION-ddl_version.sql:7]: version should be unchanged - no patches

File: groundzero_ddl/ddl_version.sql [hunk 2]
```diff
@@ -4,4 +4,4 @@ CREATE TABLE ddl_version.version (version_tag varchar(256) PRIMARY KEY, updated_
 -- Version and patch number for the current ground zero,
 -- NOTE: These must be updated every time the repo is tagged
 INSERT INTO ddl_version.patches (patch_number, applied_timestamp) VALUES (2500, current_timestamp);
-INSERT INTO ddl_version.version (version_tag, updated_timestamp) VALUES ('v4.2.0', current_timestamp);
+INSERT INTO ddl_version.version (version_tag, updated_timestamp) VALUES ('v4.3.0', current_timestamp);
```
Comment (NickGrantONS) [inline groundzero_ddl/ddl_version.sql:7]: version should be unchanged - no patches

File: patch_database.py [hunk 3]
```diff
@@ -9,10 +9,10 @@
 PATCHES_DIRECTORY_ACTION = Path(__file__).parent.joinpath('patches/action')
 
 # current_version should match the version in the ddl_version.sql file
-current_version = 'v4.2.0'
+current_version = 'v4.3.0'
```
Comment (NickGrantONS) [inline patch_database.py:12]: version should be unchanged - no patches

File: patch_database.py [hunk 4]
```diff
@@ -9,10 +9,10 @@
...
 
 # current_version_action should match the version in the ACTION-ddl_version.sql file
-current_version_action = 'v1.1.0'
+current_version_action = 'v1.2.0'
```
Comment (NickGrantONS) [inline patch_database.py:15]: version should be unchanged - no patches

**Pull request merged**